### PR TITLE
Django 1.8.4 and security

### PR DESCRIPTION
**Pull request by kamarkiewicz**
Hi!

This changes are suitable for python-3.3 cardridge, python-2.7 wasn't tested.
I updated Django version to 1.8.4 (current one) and extract SECRET_KEY to persistent file to be more secure.
I abandoned the idea of using sqlite manipulations. It's more flexible this way.

Thanks for your work.

### CODE REVIEW

**Comment by mfojtik** [comment]
@kamarkiewicz thanks a lot for this! Can you please squash the commits?

@rhcarvalho @soltysh can one of you review this PR?

**Comment by soltysh** [comment]
Will review later today

**Comment by kamarkiewicz** [comment]
@mfojtik You mean combine them all into one, lets say on another branch? No problem ;) Let me do this tomorrow

File: README.md [hunk 1]
```diff
@@ -47,20 +47,20 @@ Install the RHC client tools if you have not already done so:
...
 Create a python application
 
-    rhc app create django python-2.7
+    rhc app create django python-3.3
 
 Add this upstream repo
 
     cd django
-    git remote add upstream -m master git://github.com/openshift/django-example.git
+    git remote add upstream -m master https://github.com/kamarkiewicz/django-example.git
```
Comment (soltysh) [inline README.md]: Please don't change that do point to your repo.

File: README.md [hunk 2]
```diff
@@ -68,11 +68,10 @@ That's it. You can now checkout your application at:
...
 Admin user name and password
 ----------------------------
-As the `git push` output scrolls by, keep an eye out for a
-line of output that starts with `Django application credentials: `. This line
-contains the generated admin password that you will need to begin
-administering your Django app. This is the only time the password
-will be displayed, so be sure to save it somewhere. You might want 
-to pipe the output of the git push to a text file so you can grep for
-the password later.
+Use SSH to log into django gear. And now run this command
```
Comment (soltysh) [inline README.md]: Use `rhc ssh` to log into python gear and run this command:

File: README.md [hunk 3]
```diff
@@ -68,11 +68,10 @@ That's it. You can now checkout your application at:
...
 
+	python $OPENSHIFT_REPO_DIR/wsgi/myproject/manage.py createsuperuser
+
+And follow the steps. Now you should be able to login at
```
Comment (soltysh) [inline README.md]: ~~And follow the steps.~~ You should be now able to login at:

File: wsgi/application [hunk 4]
```diff
@@ -7,14 +7,8 @@ import sys
...
 sys.path.append(os.path.join(os.environ['OPENSHIFT_REPO_DIR'], 'wsgi', 'myproject'))
-virtenv = os.environ['APPDIR'] + '/virtenv/'
 ## GETTING-STARTED: make sure the next line has the right python version:
-os.environ['PYTHON_EGG_CACHE'] = os.path.join(virtenv, 'lib/python2.7/site-packages')
-virtualenv = os.path.join(virtenv, 'bin/activate_this.py')
-try:
-    execfile(virtualenv, dict(__file__=virtualenv))
-except:
-    pass
+os.environ['PYTHON_EGG_CACHE'] = os.path.join(os.environ['VIRTUAL_ENV'], 'lib', 'python3.3', 'site-packages')
```
Comment (soltysh) [inline wsgi/application]: Make this work with python-2.7.

**Comment by soltysh** [comment]
@kamarkiewicz generally it's [good idea](https://guides.github.com/introduction/flow/) to make fixes in a branch and then create PRs from it :wink: 
And yes we want you to create just one single commit, this is to have clean history. Generally I like what you proposed, but please make sure this works with python-2.7, I've pointed at least one place where that will be problematic.

**Comment by kamarkiewicz** [comment]
Thanks for review and your valuable comments, I will create new branch, clean this up and apply your notices and PR it.

**Pull request closed without merging**